tests/api.sipnet.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<pecan>
+  <pfts>
+    <pft>
+      <name>temperate.coniferous</name> 
+    </pft>
+  </pfts>
+
+  <meta.analysis>
+    <iter>3000</iter>
+    <random.effects>FALSE</random.effects>
+    <threshold>1.2</threshold>
+    <update>AUTO</update>
+  </meta.analysis>
+
+  <ensemble>
+    <variable>NPP</variable>
+  </ensemble>
+
+  <sensitivity.analysis>
+    <quantiles>
+      <sigma>-1</sigma>
+      <sigma>1</sigma>
+    </quantiles>
+    <variable>NPP</variable>
+  </sensitivity.analysis>
+
+  <model>
+    <type>SIPNET</type>
+    <revision>r136</revision>
+  </model>
+
+  <run>
+    <site>
+      <id>772</id>
+    </site>
+    <inputs>
+      <met>
+        <id>99000000003</id>
+      </met>
+    </inputs>
+    <start.date>2002-01-01 00:00:00</start.date>
+    <end.date>2005-12-31 00:00:00</end.date>
+    <dbfiles>pecan/dbfiles</dbfiles>
```

Comment:
per previous comment about not letting external users set paths on the host system, you probably want to take away this flag as something that's read and parsed and make sure your API is setting it correctly internally.

More broadly, you probably need to take a very close look at all the 'advanced' workflow options and think carefully about if/how we're going to allow the "power user" to to set paths when they're submitting to their own machine while sanitizing those paths out of the generic user's XML.

Reply:
I have created an env var to sort this after discussing with @robkooper